.github/workflows/build.yaml

```diff
@@ -1,7 +1,19 @@
 name: Build
 
 on:
+  workflow_call:
+    inputs:
+      useLocalPlugin:
+        description: 'Use lsp4ij locally'
+        required: true
+        type: boolean
+        default: false
   workflow_dispatch:
+    inputs:
+      useLocalPlugin:
+        description: 'Use lsp4ij locally'
+        required: true
+        default: 'false'
```

Comment:
Is there a reason this wasn't declared as a boolean like the input in the `workflow_call`?

Reply:
Initially, I had defined the workflow dispatch event as a string in the initial commits. However, it can also be declared as a boolean. I have now changed it to a boolean.